wordpress-mobile/WordPress-iOS
# [Stats Refresh] initial Post Stats view

Ref #11189 

This is the first step in creating the Post Stats view. It displays a table view with the post title card, with the post title from the selected post.

To test:

---
- On Insights `Latest Post Summary`, select `View more`.
- Verify the Post Stats view is displayed with the post title.

![lps](https://user-images.githubusercontent.com/1816888/54057141-4db06980-41af-11e9-946c-f551e1644163.png)

---
- On Period `Posts and Pages`, select a row.
- Verify the Post Stats view is displayed with the post title from the selected row.

![pandp](https://user-images.githubusercontent.com/1816888/54057257-a6800200-41af-11e9-80d6-c33aced407e1.png)
